drivers/tee/optee/supp.c

```diff
@@ -45,6 +45,7 @@ void optee_supp_uninit(struct optee_supp *supp)
 u32 optee_supp_thrd_req(struct tee_context *ctx, u32 func, size_t num_params,
 			struct tee_param *param)
 {
+	bool interruptable;
```

Comment:
seems "interruptable" is used in technical writings, and has the same meaning as 'interruptible'.